### PR TITLE
Airdrop to batch transfer

### LINKED ISSUE #26: Change airdrops to batches transfers
We are going to be moving towards using the same fraudproof as normal transfers for airdrops with changes to compression scheme and batch commitment.

### PR DESCRIPTION
**Pull request by vaibhavchellani**
Fixes #26

### CODE REVIEW

File: contracts/rollupReddit.sol [hunk 1]
```diff
@@ -5,12 +5,15 @@ import {IReddit} from "./interfaces/IReddit.sol";
 import {ParamManager} from "./libs/ParamManager.sol";
 import {Types} from "./libs/Types.sol";
 import {NameRegistry as Registry} from "./NameRegistry.sol";
+import {RollupHelpers} from "./rollup.sol";
+import {RollupUtils} from "./libs/RollupUtils.sol";
 
-contract RollupReddit {
+contract RollupReddit is RollupHelpers {
```
Comment (ChihChengLiang) [inline contracts/rollupReddit.sol:11]: What's the reason we move the submitBatch out of rollup.sol to rollup reddit?

Comment (vaibhavchellani) [reply contracts/rollupReddit.sol:11]: Oh I thought that RollupReddit is the core rollup contract, is it not?

Comment (vaibhavchellani) [reply contracts/rollupReddit.sol:11]: I thought rollup.sol was what we get from hubble-core and we are going to use rollupReddit as the coordinator frontend contract.

**Comment by vaibhavchellani** [comment]
Yeah, just saw that the tests are failing, its because I removed fraudProof.sol contract, pushing a fix!

File: contracts/rollupReddit.sol (hunk 1 above)
Comment (ChihChengLiang) [reply contracts/rollupReddit.sol:11]: Yeah, RollupReddit is the core rollup contract. Ah ok I see what we are doing here.

**Comment by ChihChengLiang** [comment]
Superseded by #40

**Pull request closed without merging**